languages/cpp/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,70 @@
+C++ is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by using the [`auto` keyword][auto] to have the C++ compiler infer its type based on the assigned value.
+
+```cpp
+int explicitVar = 10; // Explicitly typed
+auto implicitVar = 10; // Implicitly typed
+```
+
+Updating a variable's value is done through the [`=` operator][assignment-operators]. Once defined, a variable's type can never change.
+
+```cpp
+auto count = 1; // Assign initial value
+count = 2;     // Update to new value
+
+// Compiler error when assigning different type
+// count = false;
+```
+
+C++ is a multi paradigm language language and functions can be defined in a _class_ or outside. The [`class` keyword][class] is used to define a class. A [function][function] within a class is refered to as a _method_. Each [method][methods] can have zero or more parameters. All parameters must be explicitly typed, there is no type inference for parameters. Similarly the return type must also be made explicit. There are exceptions to the previous two rules but we'll talk about them in the future. Values are returned from functions using the [`return` keyword][return]. To allow a method to be called by code in other files, the `public` access modifier must be added.
+
+```cpp
+class Calculator {
+public:
+  int add(int x, int y) {
+    return x + y;
+  }
+};
+```
+
+Invoking a method is done by specifying the name of an instance of that class, called an object, the method name and passing arguments for each of the method's parameters.
+
+```cpp
+Calculator calc;
+auto sum = calc.add(1, 2);
+```
+
+If the method to be called is defined in the same class as the method that calls it, the class name can be omited.
+
+If a method does not use any class _state_ (which is the case in this exercise), the method can be made _static_ using the [`static` modifier][static].
+
+```cpp
+class Calculator {
+public:
+  static int multiply(int x, int y) {
+    return x * y;
+  }
+};
+```
+
+A static method _must_ be called directly using the class name without creating an instance:
```

Comment:
Possibly worth also explicitly calling out the use of the double semi-colon vs the period.

Reply:
Right, it's not clear right now.